backend/Tasque.Core.Common/Entities/Task.cs

```diff
@@ -11,17 +10,21 @@ public Task()
         Labels = new List<Label>();
         Users = new List<User>();
     }
-    public string Name { get; set; } = null!;
-    public string? Description { get; set; }
     public string Summary { get; set; } = null!;
-
-    public TaskState State { get; set; }
-    public TaskType Type { get; set; }
-    public TaskPriority Priority { get; set; }
+    public string? Description { get; set; }
 
     public DateTime Deadline { get; set; }
     public DateTime? FinishedAt { get; set; }
 
+    public int StateId { get; set; }
+    public TaskState State { get; set; } = null!;
```

Comment:
State, Type, Priority. It's understandable that they are related to Task.

Reply:
It's understandable when they are placed in one file. But if these files will be places in entities folder we won't be able to recognize to what do these models are related.